packages/babel-plugin-ember-test-metadata/get-normalized-file-path.js

```diff
@@ -18,10 +18,11 @@ const {
  * @returns {string} E.g. tests/acceptance/my-test.js
  */
 function getNormalizedFilePath({ packageName, getCustomNormalizedFilePath, isUsingEmbroider, projectRoot, filename, root }) {
-  if (typeof getCustomNormalizedFilePath === 'function') {
+  if (getCustomNormalizedFilePath) {
+    const customNormalizedFilePath = require(getCustomNormalizedFilePath);
     const options = { packageName, isUsingEmbroider, projectRoot, filename, root };
   
-    return getCustomNormalizedFilePath(options);
+    return customNormalizedFilePath(options);
```

Comment:
Should we verify this is a path that exists? Also, if it does exist, should we cache it for the remainder of the build?

For instance, we can do this check once, and if it's a valid path we can require the function, and cache it locally for the duration of the build.

Reply:
Actually cancel this; node will already cache the require and returned the reified module if called again within the same process.